project_status/tests/test_status.py

```
@@ -0,0 +1,30 @@
+        )
+
+    def _create_project(self):
+        project = self.env["project.project"].create(
+            {
+                "name": "Project 1",
+                "project_status": self.status.id,
+            }
+        )
+        return project
+
+    def test_01_project_status(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_01_project_status`

Add return type annotation: `None`

(ANN201)